puppetboard/static/js/environments.js

```diff
@@ -0,0 +1,11 @@
+(function () {
+  var $;
+
+  $ = jQuery;
+
+  $('#switch_env').change(function() {
+    path = location.pathname.split('/');
+    path[1] = $(this).find(':selected').text();
+    location.assign(path.join('/'))
```

Comment:
Semantic-UI supports dropdown in many places just fine. It'd be nice to have 1 framework that we use though.